Добавил в pom файл данные

psasha007 opened the pull request.

psasha007 committed:
Добавил в pom файл данные

psasha007 committed:
Добавил пакет и класс константы

psasha007 committed:
Добавил класс CreateUserAPI, запросы для пользователя

psasha007 committed:
Добавил класс CreateOrdersAPI, запросы создания заказа

psasha007 committed:
Добавил класс Credentials, для упрощенный регистрации

psasha007 committed:
Добавил класс GetOrdersFromSpecificUserAPI, запросы для получения списка заказов

psasha007 committed:
Добавил класс GettingAndUpdatingUserInfoAPI, запросы по информации о пользователе

psasha007 committed:
Добавил класс Orders body заказов

psasha007 committed:
Добавил классы Ingredients, IngredientsData тело ингредиентов

psasha007 committed:
Добавил класс CreateOrdersTest проверки создания заказа

psasha007 committed:
Добавил класс CreateUserTest проверки создания пользователя

psasha007 committed:
Добавил класс GetOrdersFromSpecificUserTest проверки получения заказов конкретного пользователя

psasha007 committed:
Добавил класс GettingAndUpdatingUserInfoTest проверки получения и обновления информации о пользователе

psasha007 committed:
Добавил класс LoginUserTest проверки Авторизации пользователя

psasha007 committed:
Добавил allure-results

nikolay-977 commented on src/main/java/CreateOrdersAPI.java:
```diff
@@ -0,0 +1,34 @@
+import io.qameta.allure.Step;
```
Можно улучшить: Хорошая практика - размещать классы в пакетах, объединяя их по функциональности (классы с запросами в src.main.java.api.client, классы с описанием модели данных в src.main.java.api.model, а в вспомогательные классы в src.main.java.api.util) либо по сущностям (src.main.java.api.courier, src.main.java.api.order)

psasha007 replied:
разместил классы в нужных пакетах

nikolay-977 commented on src/main/java/GettingAndUpdatingUserInfoAPI.java:
```diff
@@ -0,0 +1,34 @@
+// Получение и обновление информации о пользователе
+public class GettingAndUpdatingUserInfoAPI extends BaseAPI{
+    @Step("Информация о данных пользователя с авторизацией")
+    public ValidatableResponse infoUserDataWithAuthorization (String accessToken){
```
Необходимо исправить: Используй сочетание клавиш Ctrl+Alt+L (Option+Command+L для Mac) для расстановки пробелов и табуляции в соответствии с настройками Code Style в Idea. (Или в VSC: Shift + Alt + F
и Shift + Option + F для Mac)

psasha007 replied:
ага, классная штука

nikolay-977 commented on src/test/java/CreateUserTest.java:
```diff
@@ -0,0 +1,79 @@
+        Assert.assertEquals(expectedSuccess, actualSuccess);
+        Assert.assertEquals(expectedUser, actualUser);
+
+//        ValidatableResponse responseLoginUser = createUser.loginUser(Credentials.from(user));
```
Необходимо исправить: удали лишний комментированный код

psasha007 replied:
удалил лишние комментарии

psasha007 committed:
- Теперь использую хорошую практику - размещать классы в пакетах
- Спасибо, классно, теперь частенько использую сочетание клавиш Ctrl+Alt+L
- Удалил лишний комментированный код
